.github/workflows/update-sync-param-files.yaml

```diff
@@ -0,0 +1,35 @@
+name: update-sync-param-files
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+jobs:
+  sync-param-files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate-token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.PRIVATE_KEY }}
+
+      - uses: actions/checkout@v1
```

Comment:
[style]

```suggestion
      - name: Check out repository
        uses: actions/checkout@v3
```

Reply:
Please check this commit
https://github.com/autowarefoundation/autoware_launch/pull/75/commits/bd49be9fdc1248a9bd4c775e5e8769e9af44ea1c

Reply:
Thanks, but please use `v3` instead.

It's the latest version.
https://github.com/actions/checkout

Reply:
Sorry, I overlooked that one
https://github.com/autowarefoundation/autoware_launch/pull/75/commits/f985f546cbb57ace967e6b696c8f8790e8311ec6

Reply:
By the way, you can apply suggestions by clicking either one of these buttons.
![image](https://user-images.githubusercontent.com/31987104/185845819-c7e181c9-1b74-47ef-821c-ebae20e37329.png)